app/src/main/java/com/practicum/playlistmaker/App.kt

```diff
@@ -10,7 +17,14 @@ class App : Application() {
     override fun onCreate() {
         super.onCreate()
 
-        val interactor = Creator.ProvideSettingsInteractor(this)
+        startKoin {
+
+            androidContext(this@App)
+            modules(dataModule, repositoryModule, interactorModule, viewModuleModule)
+
+        }
```

Comment:
```suggestion
        startKoin {
            androidContext(this@App)
            modules(dataModule, repositoryModule, interactorModule, viewModuleModule)
        }
```

Поудаляй лишние пустые строки во всех файлах koin, читабельность сразу станет лучше